app/src/main/java/org/piramalswasthya/cho/ui/BindingUtils.kt

```
@@ -229,51 +204,11 @@ fun ViewGroup.setVisibilityOfLayout(show: Boolean?) {
-//            }
-//            form.value.value?.let { value ->
-//                children.forEach {
-//                    if ((it as RadioButton).text == value) {
-//                        clearCheck()
-//                        check(it.id)
-//                    }
-//                }
-//            }
-//        }
-//    }
 }
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)